### PR TITLE
Add documentation regarding running custom post-provision tasks

### PR DESCRIPTION
**Pull request by tzumainn**
#### What does this PR do?
It adds a small bit of documentation explaining how one might run custom post-provision tasks

#### How should this be manually tested?
No need, as this is pure documentation.

#### Is there a relevant Issue open for this?
Nope! \o/

#### Who would you like to review this?
cc: @tomassedovic @bogdando @Tlacenka  PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,43 @@ This repository contains *unsupported* code that can be used in conjunction with
...
+  - name: Add repository
+    yum_repository:
+      name: epel
+      description: EPEL YUM repo
+      baseurl: https://download.fedoraproject.org/pub/epel/$releasever/$basearch/
+```
+
+This example runs against app nodes. The list of options include:
+
+  - cluster_hosts (all hosts)
```
Comment (tomassedovic) [inline README.md]: Can you also mention `OSEv3` (which is app, infra and masters, but not dns and load balancer) and clarify that `cluster_hosts` includes dns and lb?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,43 @@ This repository contains *unsupported* code that can be used in conjunction with
 - code for provisioning various cloud providers (GCE, AWS, VMWare, [Openstack](https://github.com/openshift/openshift-ansible-contrib/tree/master/playbooks/provisioning/openstack) and [Red Hat Virtualization (RHV) / oVirt](https://github.com/openshift/openshift-ansible-contrib/tree/master/reference-architecture/rhv-ansible))
 - supporting scripts and playbooks for the various [reference architectures](https://github.com/openshift/openshift-ansible-contrib/tree/master/reference-architecture) Red Hat has published
 
+## Running Custom Post-Provision Actions
+
+If you'd like to run post-provision actions, one option is to create a custom playbook. Here's one example that adds additional YUM repositories:
```
Comment (Tlacenka) [inline README.md]: nitpick: You say _one option is to_ but then you don't introduce any other way of doing it, therefore this formulation can be a bit confusing. So either mention another way of running post-provision actions, or change the first sentence to something like this: _You can run post-provision actions by creating a custom playbook._

Comment (tzumainn) [inline README.md]: Just a note: this may be my favorite nitpick of all time : )

File: README.md (hunk 1 above)
Comment (tzumainn) [inline README.md]: Done!

**Comment by tzumainn** [comment]
Thanks for the reviews! Moved the documentation, and addressed the text feedback.

**Pull request merged**